frontend/src/pages/Home.jsx

```diff
@@ -5,8 +5,19 @@ import { Box, CardContent, Typography, Button } from "@mui/material";
 import InputBase from "@mui/material/InputBase";
 import { useForm } from "react-hook-form";
 import Grid from "@mui/material/Grid";
-import { useNavigate } from "react-router-dom";
+import { useContext, useState } from "react";
+import UserContext from "../contexts/UserContext";
+import WorkSpace from "../components/WorkSpace";
+
 export default function Home() {
+  const [message, setMesaage] = useState("");
+  const [toggle, setToggle] = useState(false);
+  const [workSpace, setWorkSpace] = useState([]);
+  const { userData, setUserData } = useContext(UserContext);
+  // userData[0].workSpace = workSpace;
+  setUserData(userData, (userData[0].workSpace = workSpace));
```

Comment:
Abi ayni sekilde yazmistim fakat sonradan aklima soyle bir soru geldi, bizim UserData'da su an tek bir user var, ama bu sayi cogaldiginda ne yapmak lazim?

Reply:
Register Page te Set ederken[...userData, {
email ,
password,
...,
...
}]

Şeklinde bir array içinde bir den fazla obje içinde kullanıcıları tutabiliriz,  fakat backend ten her bir kullanıcı için ayrı collection gelecektir, o yüzden biz bir kullanıcı imiş gibi devam edebiliriz diye düşünüyorum,  gerçi backend i yazarken nasıl yazıyorlar bilmek lazım ,benimki sadece tahmin :)